src/pages/createOrder.js

```diff
@@ -848,7 +858,7 @@ class CreateOrderPage extends React.Component {
     const { LoggedInUser } = this.props;
 
     if (!LoggedInUser) {
-      return <SignInOrJoinFree />;
+      return <SignInOrJoinFree redirect={Router.asPath} initialForm="create-account" />;
```

Comment:
We had problem in the past with `Router.asPath` (see https://github.com/opencollective/opencollective-frontend/pull/2039) it breaks because Router instance is not available in SSR. However `SignInOrJoinFree` is clever enough to use the current page for redirect if none is provided.

```suggestion
      return <SignInOrJoinFree initialForm="create-account" />;
```